src/main/resources/sql/create.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE IF NOT EXISTS "shop" (
+  "id"      SERIAL PRIMARY KEY,
+  "name"    VARCHAR(255)        NOT NULL,
+  "address" VARCHAR(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "customer" (
+  "id"      SERIAL PRIMARY KEY,
+  "name"    VARCHAR(255) NOT NULL,
+  "surname" VARCHAR(255) NOT NULL,
+  "age"     SMALLINT     NOT NULL,
+  "shop_id" INTEGER      NOT NULL,
+  FOREIGN KEY ("shop_id") REFERENCES "shop" ("id")
+);
+
+CREATE TABLE IF NOT EXISTS "purchase" (
+  "id"          SERIAL PRIMARY KEY,
+  "date"        TIMESTAMP NOT NULL,
+  "customer_id" INTEGER   NOT NULL,
+  UNIQUE ("date", "customer_id"),
```

Comment:
Field "customer_id" is the foreign key and refers to the primary key field "id" in the table "customer".
I think this field will be unique, and not necessarily indicate this.